src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -122,6 +123,16 @@ def _str_to_bool(txt: str) -> bool:
         return False
 
 
+@dataclass
+class ConfigNodeMeta:
+    key: str = ""
```

Comment:
Probably good idea to have a small docstring saying that this is "metadata for EnkfConfigNode contained in EnsembleConfig" or similar.